swan-cern/files/swan_computing_config.py

```diff
@@ -286,7 +288,9 @@ def _spark_enabled(self):
         """
 
         user_roles = self.spawner.user_roles
-        cluster = self.spawner.user_options[self.spawner.spark_cluster_field]
+        cluster = 'none'
+        if self.spawner.user_options[self.spawner.software_source] == self.spawner.lcg_special_type:
```

Comment:
Here we can also just do `cluster = self.spawner.user_options.get(self.spawner.spark_cluster_field, 'none')`